pkg/codegen/go/gen.go

```diff
@@ -826,7 +867,11 @@ func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importedPackag
 			if i == "" {
 				fmt.Fprintf(w, "\n")
 			} else {
-				fmt.Fprintf(w, "\t\"%s\"\n", i)
+				if strings.Contains(i, `"`) { // Imports with aliases already include quotes.
```

Comment:
I think you're normalizing everything to include quotes earlier; this shouldn't be necessary.

Reply:
Before this change, aliased imports would show up like this:
```
"metav1 "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/meta/v1""
```

Reply:
Yeah, I gotcha--my point was that given the code [here](https://github.com/pulumi/pulumi/pull/4096/files#diff-b02dc03ec48c179b1b6609cd77eb12a2R835), I think this call to `strings.Contains` will always return `true`, so we can simplify this to `fmt.Fprintf(w, "\t%s\n", i)`

Reply:
Double checked, and think it's correct as is. Aside from the aliased imports, which have quotes in the string already, all of the other imports (goImports + unaliased imports) don't already include quotes.

Reply:
Yeah, I see that I misread the change I pointed at earlier :)